lib/ws-kernel-picker.js

```diff
@@ -91,7 +97,9 @@ export default class WSKernelPicker {
           this.onSession.bind(this)
         );
         sessionListing.previouslyFocusedElement = this.previouslyFocusedElement;
-        sessionListing.setLoading("Loading sessions...");
+        sessionListing.selectListView.update({
+          loadingMessage: "Loading sessions..."
+        });
```

Comment:
There is a minor bug in master and in this PR where this loading message remains even after the update below. If I alt-tab away and back the updated list renders. 

Anyone know what the cause is? I'll keep working on how to fix.